15_mini_project_work/2_num_guessing_game.py

```diff
@@ -49,9 +52,11 @@ def guessing_game_run():
     guess_prompt = f'{ENTER_NUM_PROMPT}{guess_left_border}{UP_TO_SEP}{guess_right_border}{COLON_SEP} \n'
 
     while is_guess_wrong:
-        guess = input(guess_prompt)
-
-        if not is_valid(guess, guess_left_border, guess_right_border):
+        guess = input(f'{GIVE_UP_PROPOSAL}\n{guess_prompt}')
+        if guess == STOP_GAME_CHAR:
```

Comment:
STOP_GAME_RESPONSE
We might change char (1 sym) to line in the future.
Let's be more generic

Reply:
Sorry. Don't understand